here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/payload/events/QueryParameterList.java

```diff
@@ -94,6 +97,7 @@ public QueryParameterList(
     list = new ArrayList<>(16);
     UNDEFINED = new QueryParameter(this, "", -1);
     if (query_string != null) {
+      query_string = sanitize(query_string);
```

Comment:
As this is common function, I think let's not apply this fix for one specific API (related to tags), as there are different delimiters in use for other APIs.

Reply:
sure, will remove it